apps/minifront/src/components/swap/swap-form/simulate-swap-result/index.tsx

```diff
@@ -29,16 +28,22 @@ export const SimulateSwapResult = () => {
       </div>
 
       {!!traces?.length && (
-        <Box label='Routes'>
+        <>
+          <div>
+            <div className='font-bold'>Routes</div>
+            Swaps are filled in descending order of price to get you the best possible trade for
+            your tokens. Given the amount you wish to trade, here are the routes your tokens are
+            estimated to take to fulfill your swap.
+          </div>
```

Comment:
We should get rid of this text, ideally we find a way to make it comprehensible without explanation but we should avoid having to explain the UI